src/characteristic.h

```diff
@@ -17,23 +18,26 @@ typedef struct characteristic_t
 {
   char *uuid; //128-bit characteristic UUID.
   char *service_path; //Object path of the GATT service the characteristic belongs to.
```

Comment:
Is this comment still valid or does object_path fill this role now?

Reply:
The comment is still valid. A characteristic has its own object_path, and a reference to the service (service_path) which it belongs to.